### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by joaquinvanschoren**

### CODE REVIEW

**Comment by mfeurer** [comment]
Thanks, do you maybe want to update the example, too? Also, could you please redirect your PR to the development branch?

**Comment by joaquinvanschoren** [comment]
On it. 
Incidentally, don't we want to have a shorthand for sklearn, i.e. instead of doing
```
flow = flows.sklearn_to_flow(clf)
run = runs.run_flow_on_task(task, flow)
```
do something like 
```
run = runs.run_sklearn_on_task(clf, flow)
```

From what I've seen, people always use those two lines together without doing anything with the flow object.

**Comment by mfeurer** [comment]
We have [`run_model_on_task`](https://openml.github.io/openml-python/stable/generated/openml.runs.run_model_on_task.html#openml.runs.run_model_on_task).

**Comment by joaquinvanschoren** [comment]
I updated the example, but there was nothing really wrong with it?

File: doc/index.rst [hunk 1]
```diff
@@ -18,42 +18,52 @@ Example
...
-    # Set the OpenML API Key which is required to upload the runs.
-    # You can get your own API by signing up to OpenML.org.
-    openml.config.apikey = 'ABC'
     # Run the scikit-learn model on the task (requires an API key).
     run = openml.runs.run_model_on_task(task, clf)
     # Publish the experiment on OpenML (optional, requires an API key).
     run.publish()
     print('URL for run: %s/run/%d' % (openml.config.server, run.run_id))
-
+    print('View the run online: https://www.openml.org/r/%d' % run.run_id)
```
Comment (mfeurer) [inline doc/index.rst]: I think this one is redundant given the line above.

**Comment by mfeurer** [comment]
@joaquinvanschoren I did some more changes - could you please also have a look again and approve it if you like it?

**Comment by joaquinvanschoren** [comment]
Hmm, Travis fails because it can't find numpy in the Python 3.4 test. What should I do about this?

**Comment by joaquinvanschoren** [comment]
Oh oh, it seemed like Travis finished successfully, but then suddenly there was still a fail after I merged it.
The issue is that Travis fails because it can't find numpy in the Python 3.4 test. Any ideas?

**Pull request merged**